nodejs/indy/src/did/index.js

```diff
@@ -78,17 +78,20 @@ exports.getTheirEndpointDid = async function (theirDid) {
     return metadata.theirEndpointDid;
 };
 
+//TODO: replace whatever with real user password
 async function setupSteward() {
     let stewardWalletName = `stewardWalletFor:${config.walletName}`;
     try {
-        await sdk.createWallet(config.poolName, stewardWalletName);
+        await sdk.createWallet({"id": stewardWalletName}, {"key": "whatever"});
```

Comment:
Here we will also have to wait for indy-sdk 1.6 support in the node.js wrapper.

Reply:
yes

Reply:
other question... if you look in the dockerfile... I clone indy-sdk (master), build libindy and build (npm install) indy node's wrapper. After that i install the builded indy-sdk wrapper into indy-agent. The idea is to use latest version of indy-sdk wrapper and libindy... Good for development, but bad from version management point of view

Reply:
What we can do is, checkout a stable version of indy-sdk (eg 1.5) and build linindy and wrapper for this version... this is a good solution for a stable version (release) of indy-agent